PocketKit/Sources/PocketKit/Refresh/RefreshCoordinator.swift

```diff
@@ -54,7 +55,7 @@ class RefreshCoordinator {
             task.setTaskCompleted(success: false)
         }
 
-        source.refresh {
+        source.refreshSaves {
```

Comment:
do we need a `refreshArchive`

Reply:
We have one, but I dont want to always be refreshing it. I think Archive should follow an first load + on demand refresh model.

cc @nzeltzer